packages/block-editor/src/components/off-canvas-editor/block.js

```diff
@@ -145,6 +145,7 @@ function ListViewBlock( {
 
 	const { isTreeGridMounted, expand, collapse } = useListViewContext();
 
+	const isEditable = block.name !== 'core/page-list-item';
```

Comment:
We're coupling more blocks to the block editor package.

Would you consider adding a reference to this to the tracking Issue

https://github.com/WordPress/gutenberg/issues/46166

Reply:
done, thanks for thinking about this!